pysatNASA/instruments/maven_insitu.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+"""Module for the MAVEN insitu instrument.
+
+Created by: Teresa Esman, NPP at GSFC
+
```

Comment:
```suggestion
```

We've been dropping the individual author credits as per numpy standards. The commit history plus the zenodo file should handle this.